gps_sensor_broadcaster/CMakeLists.txt

```
@@ -0,0 +1,101 @@
+cmake_minimum_required(VERSION 3.16)
+project(gps_sensor_broadcaster LANGUAGES CXX)
+
+if(CMAKE_CXX_COMPILER_ID MATCHES "(GNU|Clang)")
+add_compile_options(-Wall -Wextra -Wpedantic -Werror=conversion -Werror=unused-but-set-variable
+                    -Werror=return-type -Werror=shadow -Werror=format -Werror=range-loop-construct
+                    -Werror=missing-braces)
+endif()
+
+# using this instead of visibility macros
+# S1 from https://github.com/ros-controls/ros2_controllers/issues/1053
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

See https://github.com/ros-controls/ros2_controllers/pull/1527, needs also the dependency in the package.xml of course.
```suggestion
find_package(ros2_control_cmake REQUIRED)
set_compiler_options()
export_windows_symbols()
```

---

Done